plugin-flex-ts-template-v2/src/feature-library/chat-transfer/flex-hooks/jsclient-event-listeners/conversations-client/conversationJoined.ts

```diff
@@ -2,15 +2,13 @@ import * as Flex from '@twilio/flex-ui';
 import { getWorkerFriendlyName } from '../../../utils/serverless/ChatTransferService';
 import { Worker } from 'types/task-router';
 import { Conversation } from '@twilio/conversations';
-import { isFeatureEnabled } from '../../../index';
-
+import { FlexJsClient } from "../../../../../types/feature-loader";
 
+export const clientName = FlexJsClient.conversationsClient;
+export const eventName = "conversationJoined";
```

Comment:
Would it make sense to create enums for the different event types for each JS client for consistency? Instead of manually defining the event have something like `ConversationsClient` defined in the `types/feature-loader` directory then reference them similarly to the other enums (`export const eventName = ConversationsClient.conversationJoined`)

Reply:
Good idea, I left it open because these are from non-flex SDKs, but these should be pretty straightforward to define. Maybe we can even import some enums from the SDKs themselves.